static/js/settings_users.js

```diff
@@ -392,7 +388,6 @@ exports.on_load_success = function (realm_people_data) {
         const url = '/json/users/' + encodeURIComponent(user_id) + "/reactivate";
         const data = {};
         const status = get_status_field();
-
```

Comment:
Can you undo the blank line removals here?  I think those lines were there for readability, and in any case it's distracting from the rest of the changes in this commit.